code/modules/preferences/preference_setup/skills/skills.dm

```diff
@@ -1,8 +1,39 @@
-// This file has been UNTICKED on purpose, as it is not in use.
+// TODO: this file is still commented out, because we will not even *SHOW* it until skills implementation is near critical mass.
```

Comment:
needs to be done before merge or for follow up PRs?

Reply:
todo for followup prs, i'll need to end up making an admin interface for this too